DocumentsFromSnapshotMigration/src/main/java/com/rfs/RfsMigrateDocuments.java

```diff
@@ -103,11 +103,11 @@ public static class Args {
                 + " performing the document migration.  Useful for preventing disk overflow.  Default: 80 * 1024 * 1024 * 1024 (80 GB)"), required = false)
         public long maxShardSizeBytes = 80 * 1024 * 1024 * 1024L;
 
-        @Parameter(names = { "--max-initial-lease-duration" }, description = ("Optional. The maximum time that the "
+        @Parameter(names = { "--initial-lease-duration" }, description = ("Optional. The time that the "
```

Comment:
please rename in README.md for the DocumentsFromSnapshotMigration

Reply:
Thanks!  Updated in the latest copy